required/latex-lab/testfiles-math/tagging-844.lvt

```diff
@@ -0,0 +1,18 @@
+\DocumentMetadata
+  {
+    lang=en-US,
+    pdfstandard=ua-2,
+    tagging=on,
+  }
+\input{regression-test}
+\documentclass[fleqn]{article}
+
+\begin{document}
+\START
+
+\begin{eqnarray}
+A&=&B\\
+D&=&C
+\end{eqnarray}
+
+\end{document}
```

Comment:
unless you want to record the end of doc data (like how many objects are written it is better to end with
```
\newpage
\OMIT
\end{document}
```
because then changes in the number of objects written do not alter the test output.